mail/getmail/Portfile

```diff
@@ -1,28 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 PortSystem 1.0
 PortGroup python 1.0
 
 name                getmail
-version             5.6
+version             5.13
 categories          mail python
 platforms           darwin
 maintainers         nomaintainer
 license             GPL-2
 description         extensible mail retrieval system with POP3, IMAP4, SSL support
-long_description    getmail version 4 is a flexible, extensible mail retrieval \
-                    system with support for POP3, IMAP4, SSL variants of both, \
-                    maildirs, mboxrd files, external MDAs, arbitrary message \
-                    filtering, single-user and domain-mailboxes, and many \
-                    other useful features.
+long_description    getmail version 5 is a flexible, extensible mail retrieval \
+					system with support for POP3, IMAP4, SSL variants of both, \
+					maildirs, mboxrd files, external MDAs, arbitrary message \
+					filtering, single-user and domain-mailboxes, and many \
+					other useful features.
```

Comment:
Are you using tabs here? We prefer spaces, so please go back to using spaces.

Reply:
It's not even "correct" tabs (to me, this looks like the description has been shifted dramatically to the right). Spaces are best because there's never any ambiguity about the meaning of a space, while there are arguments over 4 space tabs vs. 8 space tabs.